public/wp-content/themes/ColoredCow/landing.php

```diff
@@ -5,9 +5,88 @@
+                                    
+                                        <input type="text" class="form-control" placeholder="" name="request_name" id="request_name" maxlength="30" required>
+                                    
+                                 </div>
+                                <div class="form-group">
+                                    <label for="recipient-name" class="form-control-label">
+                                        <h6>Your Email:</h6>
+                                    </label>
+                                    
+                                        <input type="email" class="form-control" placeholder="someone@example.com" name="request_emailid" id="request_emailid" maxlength="30" required>
+                                    
+                                </div>
+                                <div class="form-group">
+                                    <label for="recipient-number" class="form-control-label">
+                                        <h6>Mobile Number:</h6>
+                                    </label>
+                                    
+                                        <input type="number" class="form-control"  placeholder="10 digit mobile no." name="phonenumber" id="phonenumber" max="9999999999" required>
+                                    
+                                </div>  
+                                <div class="form-group">
+                                    <label for="recipient-name" class="form-control-label">
+                                        <h6>Gender:</h6>
+                                    </label>
+                                    <label class="custom-control custom-radio">
+                                        <input name="request_gender" value="Male" type="radio" class="custom-control-input" required>
+                                        <span class="custom-control-indicator"></span>
+                                        <span class="custom-control-description">Male</span>
+                                    </label>
+                                    <label class="custom-control custom-radio">
+                                        <input name="request_gender" value="Female" type="radio" class="custom-control-input" required>
+                                        <span class="custom-control-indicator"></span>
+                                        <span class="custom-control-description">Female</span>
+                                    </label>
+                                </div>
+                              </div>
+                              <div class="modal-footer pull-left">
+                              <button type="button" class="btn btn-success btn-lg"  id="Submit_request">Request</button>
+                              </div>
+                        </form> 
+                    </div>
+                </div>
+            </div>
+        </div>
+        <br>
```

Comment:
let's not use <br> anymore for styling. use div's margin/padding for spacing and alignment.